.eslintrc

```diff
@@ -16,6 +16,14 @@
   ],
 
   "rules": {
+    "arrow-parens": ["error", "always"],
+    "comma-dangle": ["error", {
+      "arrays": "only-multiline",
+      "objects": "only-multiline",
+      "imports": "only-multiline",
+      "exports": "never",
+      "functions": "never"
+    }],
```

Comment:
sweet

Reply:
Doesn't look like it's needed after your config updates :+1: